### PR TITLE
Command naming conventions

### PR DESCRIPTION
**Pull request by danepowell**
This is a partial fix for https://github.com/symfony/symfony/issues/36973, to be reviewed along with https://github.com/symfony/symfony/pull/40387

### CODE REVIEW

File: contributing/code/conventions.rst [hunk 1]
```diff
@@ -234,3 +234,11 @@ of the impacted component:
 This task is mandatory and must be done in the same pull request.
 
 .. _`@-silencing operator`: https://www.php.net/manual/en/language.operators.errorcontrol.php
+
+Naming commands and options
```
Comment (fabpot) [inline contributing/code/conventions.rst]: Naming Commands and Options

File: contributing/code/conventions.rst [hunk 2]
```diff
@@ -234,3 +234,11 @@ of the impacted component:
...
+---------------------------
+
+Commands and their options should be named and described using the English
```
Comment (fabpot) [inline contributing/code/conventions.rst:241]: Commands and their arguments and options ...

**Review by fabpot — CHANGES_REQUESTED** [review]

File: contributing/code/conventions.rst (hunk 1 above)
Comment (danepowell) [reply contributing/code/conventions.rst]: The irony of screwing up styling in a style guide :) Fixed.

**Comment by javiereguiluz** [comment]
Dane, thanks for this contribution! Sorry it took us a bit to merge it. Congrats on your first Symfony Docs contribution 🎉

**Pull request merged**